.github/workflows/test-library.yml

```diff
@@ -680,13 +679,16 @@ jobs:
         docker buildx inspect
         docker buildx ls
     - name: Set PROXYPY_CONTAINER_VERSION
-      run: |
-        echo "PROXYPY_CONTAINER_VERSION=$(echo '${{ needs.pre-setup.outputs.dist-version }}' | tr + .)" > $GITHUB_ENV
+      run: >-
+        echo "PROXYPY_CONTAINER_VERSION=
+          $(echo '${{ needs.pre-setup.outputs.dist-version }}' | tr + .)"
```

Comment:
This will probably be problematic because the parser converts the line feeds to spaces in this case. So you'll get `var= <value>` — not sure if that space is safe. I like doing it a bit differently by breaking lines in less essential places:
```suggestion
        echo "PROXYPY_CONTAINER_VERSION=$(
          echo '${{ needs.pre-setup.outputs.dist-version }}' | tr + .
        )"
```